.github/workflows/deploy.yml

```diff
@@ -0,0 +1,45 @@
+name: Deploy wheels
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, windows-latest, macos-latest]
+
+    steps:
+      - uses: actions/checkout@v2
+        # Include all history and tags
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: '3.8'
+
+      - name: Install dependencies
+        run: |
+          python -m pip install cibuildwheel==1.5.1 setuptools wheel twine
```

Comment:
any good reason to fix the version?
we might forget to update for long

Reply:
No - I think it's just what they include in their [examples](https://github.com/joerick/cibuildwheel/tree/master/examples).

It shouldn't matter if we update for long though and I think we'd prefer to have reproducible runs as much as possible here.